### PR TITLE
*: cut 2.8.0

### PR DESCRIPTION
**Pull request by codesome**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +12,7 @@
 * [ENHANCEMENT] (1) Remember All/Unhealthy choice on target-overview when reloading page. (2) Resize text-input area on Graph page on mouseclick. #5201
 * [ENHANCEMENT] In `histogram_quantile` merge buckets with equivalent le values. #5158.
 * [ENHANCEMENT] Show list of offending labels in the error message in many-to-many scenarios. #5189
+* [ENHANCEMENT] Show `Storage Retention` criteria in affect in `/status` page. #5322
```
Comment (brian-brazil) [inline CHANGELOG.md]: effect on

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
-## 2.8.0-rc.0 / 2019-03-06
+## 2.8.0 / 2019-03-11
+
+This release uses WAL for remote_write API. This currently causes a little higher usage of memory, which will be improved upon in the future releases.
```
Comment (tariq1890) [inline CHANGELOG.md]: Nit: I think it would be nice to specify Write-ahead log here. Unexpanded acronyms make me very sad.

Comment (tariq1890) [inline CHANGELOG.md]:
```suggestion
This release uses WAL (write ahead log) for the remote write API. This currently causes a slight increase in memory usage which will be addressed in future releases.
```

Comment (codesome) [reply CHANGELOG.md]: No problem with that. Doing.

**Pull request merged**